nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/src/main/java/org/apache/nifi/properties/NiFiPropertiesLoader.java

```diff
@@ -289,4 +310,30 @@ private void setBootstrapKey(final SensitivePropertyProviderFactory providerFact
             }
         }
     }
+
+    private static class Props extends Properties {
```

Comment:
Recommend adjusting the class name to something like `ProcessedProperties` or `DuplicateDetectingProperties`.